x-pack/plugins/observability_solution/slo/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -123,6 +124,15 @@ export const getRuleExecutor = ({
           window: windowDef,
         } = result;
 
+        const instances = instanceId.split(',');
+        const groups =
+          instanceId !== ALL_VALUE
+            ? [slo.groupBy].flat().reduce((resultGroups: any, groupByItem, index) => {
```

Comment:
using `any` here due to same reason as mentioned in [this comment](https://github.com/elastic/kibana/pull/185933/files#r1636266048)

Reply:
Removed usage of `any` as per https://github.com/elastic/kibana/pull/185933#discussion_r1637808856 suggestion in 517b54212a936aed703bb046a58a378aa3b5ae27